### ISSUE TITLE
Expiry check fails - Mobile wallet does not identify when the expiry date is changed.

### ISSUE THREAD

**Issue by rite2bala** [PharmaLedger-IMI/epi-workspace#436]
Steps to reproduce error:

1. Create a product 
2. create a batch with valid serial number 
3. create a new 2d data-matrix with changed expiry date 

Scan the modified 2d matrix. In this case - the actual result is :
 Mobile Wallet shows the expiry date which is configured in the enterprise wallet 

Expected result:
The mobile wallet must display that the expiry date is invalid .

**Comment by adiganga2002** [comment]
This could be related to refresh period (can be changed in settings, default is 3600 sec)

**Comment by rite2bala** [comment]
@adiganga2002  - this issue persists. 
I can show a demo for this when you are available 

Steps followed: 

1. We changed the refresh time to 10 seconds on mobile wallet
2. Created new product on Enterprise wallet
3. Created new batch on Enterprise wallet
4. Scanned the batch on the mobile wallet 
5. Changed the expiration date  on enterprise wallet 
6. scanned the old 2d matrix from step 4 again 
App is unable to bring up incorrect expiration date error.
